lib/kitchen/directions/bake_notes/bake_numbered_notes.rb

```diff
@@ -7,6 +7,9 @@ def self.v1(book:, classes:)
         classes.each do |c|
```

Comment:
Let's change the `c` to `klass` -- fyi @KarinaMendez2 I told @kswanson33 I'm not a fan of single letter variable names -- they are hard to search for and hard to understand when reading code that uses them ("what's 'c' in this line?")